ui/src/app/app-routing.tsx

```diff
@@ -23,11 +24,10 @@ const signInGuard: Guard = {
   redirectPath: '/login'
 };
 
-// This will be added back in after the protectedRoutes fix
-// const registrationGuard: Guard = {
-//   allowed: (): boolean => hasRegisteredAccess(profileStore.get().profile.dataAccessLevel),
-//   redirectPath: '/'
-// };
+const registrationGuard: Guard = {
+  allowed: (): boolean => hasRegisteredAccess(profileStore.get().profile.dataAccessLevel),
```

Comment:
Existing issue, but we're not treating this as state, right? so there is nothing to cause a route to be rerendered as a result of a registration status change, for example?

I think this issue is purely hypothetical for now, since Angular routing should cause this to be reevaluated on every navigate anyways.

Reply:
That is correct. The guards are based on stores at the moment. We could use a hook in the top level router to subscribe to the store and re-render on update.